docker-compose.yaml

```diff
@@ -97,8 +99,7 @@ services:
     command: |
       bash -xec "
         if [[ ! -f ./node_modules/@grafana/toolkit/bin/grafana-toolkit.js ]]; then npm install --production=false; fi && \
-        source /grafana-clickhouse/.release_env && \
-        node ./node_modules/@grafana/toolkit/bin/grafana-toolkit.js plugin:sign
+        npm run sign
```

Comment:
Removed duplicate maintenance for this command

Reply:
nice catch thanks